kie-server-parent/kie-server-api/src/test/java/com/example/manyrules/JSONMarshallerLargeTest.java

```diff
@@ -0,0 +1,42 @@
+package com.example.manyrules;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.drools.core.util.IoUtils;
+import org.junit.Test;
+import org.kie.server.api.marshalling.json.JSONMarshaller;
+import org.kie.server.api.marshalling.test.model.Fact;
+import org.reflections.Reflections;
+import org.reflections.scanners.SubTypesScanner;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assume.assumeThat;
+
+public class JSONMarshallerLargeTest {
+
+    private Logger logger = LoggerFactory.getLogger(JSONMarshallerLargeTest.class);
+
+    @Test(timeout=5000L)
+    public void testLargeNumberOfClasses() throws IOException {
+        assumeThat(JSONMarshaller.isDefaultTypingActivate(), is(false));
```

Comment:
We should run the test by default with `org.kie.server.json.defaultTyping.enabled=false` and not skip it

Reply:
done